web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -889,7 +896,7 @@ export default defineComponent({
             v-on="on"
           >
             <v-btn
-              color="primary"
+              v-bind:color="type==='canSubmit' ? 'primary' : 'green'"
```

Comment:
What is `type` here meant to be? I see some errors in my console on this branch related to it not being available here. I think this could just be

```suggestion
              color="green"
```
Side note: there is a [shorthand](https://vuejs.org/api/built-in-directives.html#v-bind) for `v-bind`, which is what we opt for in this code base.

Reply:
I'm not actually sure what the `type` variable is there either, I took the syntax from a tutorial for changing the color based on a variable. 
The intent for this line was to swap the color to green once a user is able to click the submit button. Does changing the color to green without the v-bind still accomplish that?

Reply:
My suggestion will make the Submit button always appear as the "green" color. I'm not sure we'd need to have a different color if the user was unable to submit. We also use the "disabled" prop which should add additional styling to indicate that the user can't submit.

If you wanted to change color depending on the value of a variable, and the variable is already a boolean, you can just use that in the ternary operator:
```
:color="canSubmit ? 'green' : 'primary'"
```
Note the use of the shorthand for `v-bind` here. Vue will evaluate the expression in `""` and use that value for `color`. If `canSubmit` is `true`, the color will evaluate to `green`, else `primary`.

Reply:
Ah ok that makes sense. I think with the "disabled" prop it's fine to leave it green like you said. I'll make that change. Thanks for explaining everything, it's very helpful to know the rationale.